### PR TITLE
Switching example app from Stytch.Example SDK to Stytch.net SDK

### PR DESCRIPTION
**Pull request by ashWeaver-Stytch**
switching from Example.Stytch to Stytch.net

### CODE REVIEW

File: stytch-react-dotnet-example.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (ollie-stytch) [inline stytch-react-dotnet-example.sln:2]: This seems like something that should be in the .gitignore as well `*.sln`

**Comment by ollie-stytch** [comment]
Let's make the title of the PR more descriptive.

**Review by ollie-stytch — APPROVED** [review]

File: stytch-react-dotnet-example.sln (hunk 1 above)
Comment (ashWeaver-Stytch) [reply stytch-react-dotnet-example.sln:2]: adding!

**Review by ashWeaver-Stytch — COMMENTED** [review]
Made changes to ignore .sln files and removed the .sln file from the pull request

**Pull request merged**